source/extensions/filters/common/expr/context.h

```diff
@@ -95,6 +96,19 @@ constexpr absl::string_view ListenerMetadata = "listener_metadata";
 constexpr absl::string_view ListenerDirection = "listener_direction";
 constexpr absl::string_view Node = "node";
 
+// Stream duration properties
+constexpr absl::string_view StreamDuration = "stream_duration";
```

Comment:
Do we really need all of these? What's the actual measurement you are looking for?

Reply:
We need calculate the overall latency, the backend latency and the proxy latency. How they are calculated is documented below.

https://docs.google.com/document/d/1724JXCXRgvxAWvnxyZwg-cqBRHg11f_rtwzfBFg1V4w/edit?tab=t.0#heading=h.l9t2qgm5471t

Reply:
Updated to backend_latency. PTAL